src/tensors/cpu/expression_graph_packable.h

```diff
@@ -1,10 +1,57 @@
 #pragma once
 
 #include "graph/expression_graph.h"
-#include "packed_gemm.h"
+#include "fbgemm/packed_gemm.h"
+#include "tensors/cpu/integer_common.h"
+
+namespace { //Temporary annonymous transposition, until we figure out how to access the proper one
+inline void transpose4x4_SSE(const float* A,
```

Comment:
Can we move this to integer_common somewhere?

Reply:
This is a shameless code duplication, because I couldn't figure out how to access a transpose outside of the graph. Can we do this in a better way? I took this one from `tensors/cpu/tensor_operators.cpp`